packages/forms/frontend/sirius-components-widget-reference/src/components/ValuedReferenceAutocomplete.tsx

```diff
@@ -271,20 +247,9 @@ export const ValuedReferenceAutocomplete = ({
             label={option.label}
             data-testid={`reference-value-${option.label}`}
             icon={
-              option.iconURL.length > 0 && (
-                <div className={classes.iconContainer}>
-                  {option.iconURL.map((icon, index) => (
-                    <img
-                      height="16"
-                      width="16"
-                      key={index}
-                      alt={option.kind}
-                      src={httpOrigin + icon}
-                      className={classes.icon}
-                      style={{ zIndex: index }}></img>
-                  ))}
-                </div>
-              )
+              <div>
```

Comment:
Is the `div` necessary?

Reply:
Yes, MUI add some chip style related properties to the div only if the div is at this level.